common/setups/rasr/hybrid_system.py

```diff
@@ -384,15 +398,31 @@ def nn_recognition(
 
             for pron, lm, prior, epoch in itertools.product(pronunciation_scales, lm_scales, prior_scales, epochs):
                 assert epoch in checkpoints.keys()
-                assert acoustic_mixture_path is not None
-
-                if use_epoch_for_compile:
-                    tf_graph = self.nn_compile_graph(name, returnn_config, epoch=epoch)
+                acoustic_mixture_path = CreateDummyMixturesJob(
```

Comment:
This will not work for two reasons: First this always overwrites `acoustic_mixture_path` no matter the input. Secondly the temporary variable is needed because otherwise nesting in python overwrites `acoustic_mixture_path` for the whole system.